avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml

```diff
@@ -27,7 +26,6 @@ spec:
     - Azure.ContainerApp.PublicAccess
     # Azure Virtual Machine
     - Azure.VM.AMA
-    - Azure.VM.Standalone
```

Comment:
On this I'd like @rahalan to comment as I remember he discussed a reason for this to be in place. If so let's maybe add a comment to detail that reason

Reply:
This would just require in the default test to have an AZ specified.

I will be submitting another test for the VM module to check that the AZ parameter does not have a default value very soon, as this is what we agreed recently and a consumer must provide inputs for the AZ parameter

Reply:
Agree, just wanted to double check with the module owner if the exception is there for any other reason or put just temporarily to speed up the migration (I guess the latter)

Reply:
@jtracey93 @eriqua the reason was, that aa AZ for just one VM makes no sense and even may create a feeling of better SLA, that is not provided.